src/main/kotlin/ch/hevs/cloudio/cloud/services/InfluxUpdateSetService.kt

```diff
@@ -23,12 +30,42 @@ class InfluxUpdateSetService(val env: Environment, val influx: InfluxDB) : Abstr
     //get database to write by environment property, has default value
     val database: String by lazy { env.getProperty("CLOUDIO_INFLUX_DATABASE", "CLOUDIO") }
 
+    val pointsSet: MutableSet<Point> = mutableSetOf()
+
+    val influxBatchEnable by lazy { env.getProperty("cloudio.influxBatchEnable", "false").toBoolean() }
+    val influxBatchTimeMs : Long by lazy { env.getProperty("cloudio.influxBatchIntervallMs", "1000").toLong() }
+
+
     @PostConstruct
     fun initialize()
     {
         // Create database if needed
         if(influx.query(Query("SHOW DATABASES","")).toString().indexOf(database)==-1)
             influx.query(Query("CREATE DATABASE $database",""))
+
+        try{
+            if(influxBatchEnable) {
```

Comment:
The InfluxDB client has build-in batch support, we should use that:
https://influxdata.github.io/influxdb-client-java/influxdb-client-java/apidocs/index.html

Reply:
I'm already using BatchPoints provided by influx api and that's how I send data to influx if the user has set cloudio.influxBatchEnable at true.

Reply:
I do not understand why there is this timer, the InfluxDB object has this functionality (flush time) AFAIK.

Reply:
The goal is to push a batch of point every _cloudio.influxBatchIntervallMs_ ms an not send each 
point one by one

Reply:
If I did not misunderstood the idea you want to achieve that the points are written to the database each n milliseconds.

The InfluxDB client has an option to either write after n points or at least after n milliseconds if the point limit was not reached. The following example tries to write always 2000 points in one request but after 10 seconds the points are written even if there are not 2000 points ready yet:

```java
influxDB.enableBatch(BatchOptions.DEFAULTS.actions(2000).flushDuration(10000));
```

Reply:
> 
> ```java
> influxDB.enableBatch(BatchOptions.DEFAULTS.actions(2000).flushDuration(10000));
> ```

I'll try to implement that, thanks for the infos

Reply:
@cm0x4D  should I let the _cloudio.influxBatchEnable_ or we have by default the batch enabled? (I tested enableBatch and it works great)